src/VoipUserMapper.ts

```diff
@@ -42,13 +42,20 @@ export default class VoipUserMapper {
         return results[0].userid;
     }
 
-    public async getOrCreateVirtualRoomForRoom(roomId: string): Promise<string> {
+    private async getVirtualUserForRoom(roomId: string): Promise<string> {
         const userId = DMRoomMap.shared().getUserIdForRoomId(roomId);
         if (!userId) return null;
 
         const virtualUser = await this.userToVirtualUser(userId);
         if (!virtualUser) return null;
 
+        return virtualUser;
+    }
+
+    public async getOrCreateVirtualRoomForRoom(roomId: string): Promise<string> {
```

Comment:
Return type is `string` or `null`

fwiw, there's an `Optional<T>` type from the events-sdk that I've been littering on cases like this.

Reply:
I thought the thing to do longer-term was to switch to undefineds so we can use typescript's built in optional?

Reply:
it should boil down to the same thing